manifests/sysmgmt.yaml

```diff
@@ -78,7 +78,7 @@ spec:
     namespace: services
   - name: cray-cfs-operator
     source: csm-algol60
-    version: 1.17.0-beta.4
+    version: 1.16.2
```

Comment:
Is that expected that `cray-cfs-operator` and `cfs-hwsync-agent` versions are decreased?

Reply:
Yes, one of our developers was incorrectly tagging the development branch with beta tags. This is software that was not released and the versions were not correct, so the decrease in version is okay.

Reply:
Fixed.